modules/ipfs/src/internal/helpers.ts

```diff
@@ -164,11 +164,11 @@ export namespace Helpers {
   }
 
   function encodeParams<T>(
-    options: T,
+    options: ArrayLike<T> | {}
```

Comment:
Why is this type changing? The usages I see above use it as an object, not as an array